src/algebra/lattice_ordered_group.lean

```diff
@@ -125,26 +125,35 @@ calc a⊓b * (a ⊔ b) = a ⊓ b * ((a * b) * (b⁻¹ ⊔ a⁻¹)) :
 
 namespace lattice_ordered_comm_group
 
--- Bourbaki A.VI.12 Definition 4
 /--
+Let `α` be a lattice ordered commutative group with identity `1`. For an element `a` of type `α`,
+the element `a ⊔ 1` is said to be the *positive component* of `a`, denoted `a⁺`.
+-/
+@[to_additive /-"
 Let `α` be a lattice ordered commutative group with identity `0`. For an element `a` of type `α`,
 the element `a ⊔ 0` is said to be the *positive component* of `a`, denoted `a⁺`.
--/
-@[to_additive pos
-  "Let `α` be a lattice ordered commutative group with identity `0`. For an element `a` of type `α`,
-  the element `a ⊔ 0` is said to be the *positive component* of `a`, denoted `a⁺`."]
-def mpos (a : α) : α :=  a ⊔ 1
-postfix `⁺`:1000 := mpos
+"-/,
+priority 100
+] -- see Note [lower instance priority]
+instance has_one_lattice_has_pos_part : has_pos_part (α)  := ⟨λa, a ⊔ 1⟩
+
+@[to_additive pos_part_def]
+lemma m_pos_part_def (a : α) : a⁺ = a ⊔ 1 := by unfold has_pos_part.pos
```

Comment:
```suggestion
lemma pos_part_def (a : α) : a⁺ = a ⊔ 1 := rfl
```
what does the `m` indicate?

Reply:
(same for `neg_part`)

Reply:
The trouble is, if I do something like:
```
@[to_additive]
lemma neg_part_def (a : α) : a⁻ = a⁻¹ ⊔ 1 := rfl
```
we get an error:
```
to_additive: can't transport lattice_ordered_comm_group.neg_part_def to itself.
Give the desired additive name explicitly using `@[to_additive additive_name]`. 
```
because there's nothing in the lemma name that `to_additive` knows how to change to get a unique name for the additive form.

Since in practice this theory will almost always be used in the additive case (typically a vector space) the convention recommended by @jcommelin here https://github.com/leanprover-community/mathlib/pull/8673#discussion_r693308231 was that multiplicative lemmas be prefixed with `m_` and the additive name be manually specified with no prefix.

Reply:
I've committed and pushed the shorter proofs you suggested - thanks.